docs/grafana.md

```diff
@@ -0,0 +1,139 @@
+# Grafana
+
+---
+
+!!! note
+
+    This deployment makes a few assumption:
+        * assumes you are using OAuth using Azure
+        * assumes you are using tls/ssl
+        * assumes you are using ingress
+    If this does not apply to your deployment adjust the overrides.yaml file and skip over any unneeded sections here
+
+## Create secret client file
+
+In order to avoid putting sensative information on the cli, it is recommended to create and use a secret file instead.
+
+example secret file:
+
+```
+apiversion: v1
+data:
+  client_id: base64_encoded_client_id
+  client_secret: base64_encoded_client_secret
+kind: secret
+metadata:
+  name: azure-client
+  namespace: grafana
+type: opaque
+```
+
+---
+
+## Create a datasources yaml
+
+If you have specific datasources that should be populated when grafana deploys, create a seperate datasource.yaml.  The example below shows one way to configure prometheus and loki datasources.
+
+example datasources yaml file:
+
+```
+datasources:
+  datasources.yaml:
+    apiversion: 1
+    datasources:
+    - name: prometheus
+      type: prometheus
+      access: proxy
+      url: http://kube-prometheus-stack-prometheus.prometheus.svc.cluster.local:9090
+      isdefault: true
+    - name: loki
+      type: loki
+      access: proxy
+      url: http://loki-gateway.{{ .release.namespace }}.svc.cluster.local:80
+      editable: false
+```
+
+---
+
+## Create your ssl files
+
+If you are configuring grafana to use tls/ssl, you should create a file for your certificate and a file for your key.  After the deployment, these files can be deleted if desired since the cert and key will now be in a Kubernetes secret.
+
+Your cert and key files should look something like the following (cert and key example taken from [VMware Docs](https://docs.vmware.com/en/VMware-NSX-Data-Center-for-vSphere/6.4/com.vmware.nsx.admin.doc/GUID-BBC4804F-AC54-4DD2-BF6B-ECD2F60083F6.html "VMware Docs")).
+
+example cert file:
```

Comment:
Here I'd use the admonishment for example 

``` 
!!! example 

    ...

```

Reply:
New commit pushed with requested change.